src/prerender.ts

```diff
@@ -81,6 +81,12 @@ export async function prerender (nitro: Nitro) {
       }
     }
 
+    // Skip route
+    if (_route.skip) { return }
```

Comment:
When the route is skipped, I expect also not crawling it's children :)

Reply:
Updated